skopt/utils.py

```diff
@@ -292,3 +295,44 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_gpdims(dimensions):
```

Comment:
I am with you. Just wanted to make sure that I wasn't missing any other places that could use the dimension transformations. 

I have updated the code. Now `transform_dims` has an argument `estimator="GP"`. I figured a default setting to the Gaussian process would work best.